lab-andrew/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "lab-andrew",
+  "version": "0.1.0",
+  "description": "cowsay server req/res",
+  "main": "server.js",
+  "scripts": {
+    "test": "gulp-eslint"
+  },
+  "author": "andrewpeterwatson@gmail.com",
+  "license": "ISC",
+  "devDependencies": {
+    "gulp-eslint": "^2.0.0"
```

Comment:
You'll need to add 'gulp' to your devDependencies since your gulpfile requires it to run.  You'll also need to add 'cowsay' to your dependencies because your server cannot run without it.